azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -1493,89 +1490,85 @@ private void handleJobPage(final HttpServletRequest req, final HttpServletRespon
         return;
       }
       if (!hasPermission(project, user, Type.READ)) {
-        throw new AccessControlException("No permission to view project "
-            + projectName + ".");
+        throw new AccessControlException("No permission to view project " + projectName + ".");
       }
-
       page.add("project", project);
-      flow = project.getFlow(flowName);
+
+      final Flow flow = project.getFlow(flowNodePath);
```

Comment:
It appears that each code block fetches some info and then populates the page object. Can you add 1 liner comment before each of them?